webapp/components/wpt-runs.html

```diff
@@ -178,6 +195,24 @@
         }
         return 'present';
       }
+
+      runList(testRuns, browser) {
+        return testRuns[browser] || [];
+      }
+
+      runLink(run) {
+        let link = `/results?sha=${run.revision}`;
+        for (const label of ['experimental', 'stable']) {
+          if (run.labels && run.labels.includes(label)) {
+            link += `&label=${label}`;
```

Comment:
OK, this is the bit that you mentioned could add all labels to URLs. That's probably not going to be an improvement since it might still match multiple runs, but just a note that using https://url.spec.whatwg.org/#urlsearchparams would make the code for it quite nice at least.

Reply:
Done.